package.json

```diff
@@ -196,33 +196,33 @@
             "view/title": [
                 {
                     "command": "jfrog.xray.connect",
-                    "when": "jfrog.connection.status != signedIn",
-                    "group": "navigation@0"
+                    "when": "jfrog.connection.status != signedIn && view =~ /^jfrog.issues$|^jfrog.xray.ci.issues$/",
+                    "group": "navigation@1"
```

Comment:
It looks like the `&& view =~ /^jfrog.issues$|^jfrog.xray.ci.issues$/` alone solved the problem. Is there a reason for changing the group navigations?